Formula/kafka.rb

```diff
@@ -44,7 +37,7 @@ def install
     libexec.install "libs"
 
     prefix.install "bin"
-    bin.env_script_all_files(libexec/"bin", Language::Java.java_home_env("1.8"))
+    bin.env_script_all_files(libexec/"bin", Language::Java.java_home_env("11"))
```

Comment:
You're depending on 15 here. @jonchang @bayandin  is there an easier way?

Reply:
You can just use `Language::Java.java_home_env` and I think it uses the latest version.